### PR TITLE
✨ Forta guide v2

### PR DESCRIPTION
**Pull request by cairoeth**

### CODE REVIEW

File: docs/modules/ROOT/pages/guide/balance-automation-forta.adoc [hunk 1]
```diff
@@ -0,0 +1,335 @@
...
+
+[[pre-requisites]]
+== Pre-requisites
+
+* OpenZeppelin Defender 2.0 account. You can sign up to Defender 2.0 https://defender.openzeppelin.com/v2/?utm_campaign=Defender_2.0_2023&utm_source=Docs#/auth/sign-up[here, window=_blank].
+
+[[install-dependencies]]
+== 1. Install Dependencies
+
+NOTE: Although this guide makes use of the `defender-client` packages to create a Relayer, Monitor, and Action, the very same functionality is also available via the Defender web interface.
```
Comment (shahnami) [inline docs/modules/ROOT/pages/guide/balance-automation-forta.adoc:15]: Should this be defender-sdk?

File: docs/modules/ROOT/pages/guide/balance-automation-forta.adoc [hunk 2]
```diff
@@ -0,0 +1,335 @@
...
+
+You'll need to install the relevant Defender NPM packages. Note that Forta bot creation also requires that you also have https://www.docker.com/get-started[Docker] installed.
+
+```
+$ mkdir balance-automation && cd balance-automation
+$ npm init -y
+$ npm i --save-dev @openzeppelin/defender-relay-client @openzeppelin/defender-autotask-client @openzeppelin/defender-sentinel-client dotenv
```
Comment (shahnami) [inline docs/modules/ROOT/pages/guide/balance-automation-forta.adoc:22]: Same here, we should probably refer to defender-sdk

**Review by shahnami — APPROVED** [review]
LGTM, however, we should consider referencing defender-sdk over defender-client.

**Comment by cairoeth** [comment]
Updated to use `defender-sdk`

**Review by shahnami — APPROVED** [review]

**Pull request merged**